tests/test_loggers.py

```diff
@@ -17,114 +17,123 @@
     BytesLoggerFactory,
     PrintLogger,
     PrintLoggerFactory,
+    WriteLogger,
+    WriteLoggerFactory,
 )
 
 from .utils import stdlib_log_methods
 
 
-class TestPrintLogger:
-    def test_prints_to_stdout_by_default(self, capsys):
+class TestLoggers:
+    """Tests common to the Print and WriteLoggers."""
+
+    @pytest.fixture(name="logger_cls", params=(WriteLogger, PrintLogger))
+    @staticmethod
+    def logger_cls(request):
```

Comment:
```suggestion
    def _logger_cls(request):
```

The point of `name=` is to not have the fixture name in the namespace and accidentally using it.